src/ui/FormWidgets/DatePicker.ts

```diff
@@ -84,7 +86,14 @@ export class DatePicker implements IFormWidget, IFormWidgetSettable {
     this.element.readOnly = true;
     this.picker = new Pikaday({
       field: this.element,
-      onSelect: this.onChange
+      onSelect: this.onChange,
+      i18n: {
```

Comment:
i18n is just the option that pikaday (an external library) accept. Not my choice/decision.

I'm passing it both localization data (from `l`) and internationalization data (from `globalize` )